cmd/livepeer_bench/livepeer_bench.go

```diff
@@ -0,0 +1,216 @@
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	//"runtime/pprof"
+
+	"github.com/livepeer/go-livepeer/common"
+	"github.com/livepeer/lpms/ffmpeg"
+	"github.com/livepeer/m3u8"
+)
+
+func main() {
+	/*
+		cprof, err := os.Create("bench.prof")
+		if err != nil {
+			panic(err)
+		}
+		pprof.StartCPUProfile(cprof)
+		defer pprof.StopCPUProfile()
+	*/
+	// Override the default flag set since there are dependencies that
+	// incorrectly add their own flags (specifically, due to the 'testing'
+	// package being linked)
+	flag.Set("logtostderr", "true")
+	flag.CommandLine = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
+
+	fname := flag.String("fname", "", "Input m3u8 manifest file")
+	conc := flag.Int("conc", 1, "# of concurrent transcode sessions")
+	segs := flag.Int("segs", 0, "Maximum # of segments to transcode (default all)")
+	profs := flag.String("profs", "P240p30fps16x9,P360p30fps16x9,P720p30fps16x9", "Transcoding options for broadcast job, or path to json config")
+	nvidia := flag.String("nvidia", "", "Comma-separated list of Nvidia GPU device IDs to use for transcoding")
+	outPrefix := flag.String("outPrefix", "", "Output segments' prefix (no segments are generated by default)")
+
+	flag.Parse()
+
+	if *fname == "" {
+		panic("Please provide the input manifest as `-fname <input.m3u8>`. See -h or -help for more.")
```

Comment:
Use `glog.Fatal()` or `glog.Fatalf()` (see [docs](https://godoc.org/github.com/golang/glog#Fatalf)) to match the usage in livepeer.go? Generally, I think it is preferable to use Fatal/Fatalf to handle user input errors and to only use panic for oh-no-something-is-really-wrong errors where the stack trace might be necessary for debugging. This isn't a problem in this file right now, but one downside of using Fatal/Fatalf is that defer statements won't be executed.